src/main/java/reposense/frontend/RepoSense.java

```diff
@@ -15,10 +17,19 @@
 
     private static final Logger logger = LogsManager.getLogger(RepoSense.class);
 
+    private static void setDatesToRepoConfigs(List<RepoConfiguration> configs,
+                                              Optional<Date> sinceDate, Optional<Date> untilDate) {
```

Comment:
This static method can probably be shifted to `RepoConfiguration` or, if there are many such methods, create a `ConfigurationUtil`.